src/utils/Util.js

```diff
@@ -52,5 +52,61 @@ module.exports = {
     } catch (err) {}
     return settingsjson;
   },
+  isOfficialRepo: function (name) {
+    if (!name || !name.length) {
+      return false;
+    }
+    var repoRegexp = /^[a-z0-9]+(?:[._-][a-z0-9]+)*$/;
```

Comment:
Why does this regular expression distinguish an official repo from a non-official one? At least a comment is needed.